src/main/java/Duke.java

```diff
@@ -1,10 +1,269 @@
+                DIVIDER
+                        + " Welcome.\n"
+                        + " I am your assistant Friday. ✧ʕ̢̣̣̣̣̩̩̩̩·͡˔·ོɁ̡̣̣̣̣̩̩̩̩✧\n"
+                        + " Just FYI, I am developed by Song Yu.\n"
+                        + " May I know what I can help you?\n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints greetings when the program ends.
+     */
+    public static void printExitGreetings() {
+        System.out.print(
+                DIVIDER
+                        + " Thank you for getting in touch.\n"
+                        + " See you next time.\n"
+                        + "✧( ु•⌄• )◞ᴴᴬᵛᴱ ᴬ ᴳᴼᴼᴰ ᵀᴵᴹᴱ\n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints the error message when user's input does not match any command.
+     */
+    public static void printNotCommand() {
+        System.out.println(
+                DIVIDER
+                        + " Sorry, your command is not recognized\n"
+                        + " See 'help'. \n"
+                        + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Prints the help message.
+     */
+    public static void printHelp() {
+        String helpMessage = " 'help'     : Display tips on using this application\n"
+                + " 'exit'     : Exit the application\n"
+                + " 'bye'      : Exit the application\n"
+                + " 'list'     : List all type of tasks you added to your task list\n"
+                + " 'done'     : Mark a task as done\n\t"
+                + " e.g. 'done 2' will mark the second task as done\n"
+                + " 'todo'     : Add a new todo task\n\t"
+                + " e.g. 'todo read book' will add \"read book\" to your task list\n"
+                + " 'deadline' : Add a new deadline task\n\t"
+                + " e.g. 'deadline read book /by Sunday' will"
+                + " add \"read book\" to your task list with deadline \"Sunday\"\n"
+                + " 'event' : Add a new event task\n\t"
+                + " e.g. 'event read book /at Mon 2-4pm' will"
+                + " add \"read book\" to your task list with period  \"Mon 2-4pm\"\n";
+
+        System.out.println(
+                DIVIDER_LINE_ONLY + DIVIDER
+                        + helpMessage
+                        + DIVIDER_LINE_ONLY + DIVIDER_LINE_ONLY
+        );
+    }
+
+    /**
+     * Gets command from user input.
+     *
+     * @param userInput user's input.
+     * @return command.
+     */
+    private static String getCommand(String userInput) {
+        String[] inputWords = userInput.split(" ");
+        return inputWords[0];
+    }
+
+    /**
+     * Gets user input and execute corresponding command until the loop is exit.
+     *
+     * @param sc       Java Scanner to get user input.
+     * @param taskList The list of Task objects.
+     */
+    private static void loopOperation(Scanner sc, TaskList taskList) {
```

Comment:
Instead of using nouns such as loopOperation as the name of your method, perhaps you could use verbs like: operateLoop(). This would make it more understandable for the readers.

Reply:
Noted. Thank you